build.gradle.kts

```diff
@@ -14,3 +14,14 @@ tasks.withType<Wrapper> {
 
     finalizedBy(subprojectsWrapper)
 }
+
+tasks.register<Exec>("installGitHooks") {
+    group = "Build Setup"
+    description = "Install local repository git hooks"
+    commandLine("git")
+    args("config", "core.hooksPath", ".git_hooks")
+    onlyIf { !project.hasProperty("ci") }
+}
```

Comment:
If you found them not convenient, please let me know.
We could make individual checks opt-in \ opt-out.